src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  width: 100%;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: block;
+  color: black;
+  text-decoration: none;
+  line-height: 60px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
Move repeatable colors into variables

Reply:
Not exactly understand what I need to do

Reply:
That means you have to create [css variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) for colors you use more than once

Reply:
Not fixed 
<img width="849" alt="image" src="https://user-images.githubusercontent.com/99403198/233613090-3b995a5f-fe9e-439b-9a68-81c70af133ea.png">